Picked Local content provider.
Using local repo /tmp/r2d-test-2020-07-23T11.46jc68w0o1/github.com/agconti/dog-walks.
Using PipfileBuildPack builder
Step 1/54 : FROM buildpack-deps:bionic
 ---> 5d109dc6d8a0
Step 2/54 : ENV DEBIAN_FRONTEND=noninteractive
 ---> Using cache
 ---> 84875aa039ff
Step 3/54 : RUN apt-get -qq update &&     apt-get -qq install --yes --no-install-recommends locales > /dev/null &&     apt-get -qq purge &&     apt-get -qq clean &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 2c6f3445209c
Step 4/54 : RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen &&     locale-gen
 ---> Using cache
 ---> 5ac40a1069ef
Step 5/54 : ENV LC_ALL en_US.UTF-8
 ---> Using cache
 ---> 0569ecd6efb3
Step 6/54 : ENV LANG en_US.UTF-8
 ---> Using cache
 ---> bd3372a8f400
Step 7/54 : ENV LANGUAGE en_US.UTF-8
 ---> Using cache
 ---> ab87a637d837
Step 8/54 : ENV SHELL /bin/bash
 ---> Using cache
 ---> 50ed7363eaf5
Step 9/54 : ARG NB_USER
 ---> Using cache
 ---> 0b8893bd6422
Step 10/54 : ARG NB_UID
 ---> Using cache
 ---> b9e6fa4ea16e
Step 11/54 : ENV USER ${NB_USER}
 ---> Using cache
 ---> b142a4c01942
Step 12/54 : ENV HOME /home/${NB_USER}
 ---> Using cache
 ---> e11d5723ab90
Step 13/54 : RUN groupadd         --gid ${NB_UID}         ${NB_USER} &&     useradd         --comment "Default user"         --create-home         --gid ${NB_UID}         --no-log-init         --shell /bin/bash         --uid ${NB_UID}         ${NB_USER}
 ---> Using cache
 ---> e20a5903ae77
Step 14/54 : RUN wget --quiet -O - https://deb.nodesource.com/gpgkey/nodesource.gpg.key |  apt-key add - &&     DISTRO="bionic" &&     echo "deb https://deb.nodesource.com/node_10.x $DISTRO main" >> /etc/apt/sources.list.d/nodesource.list &&     echo "deb-src https://deb.nodesource.com/node_10.x $DISTRO main" >> /etc/apt/sources.list.d/nodesource.list
 ---> Using cache
 ---> 730bb79c819d
Step 15/54 : RUN apt-get -qq update &&     apt-get -qq install --yes --no-install-recommends        less        nodejs        unzip        > /dev/null &&     apt-get -qq purge &&     apt-get -qq clean &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 2be5b9d2edc7
Step 16/54 : EXPOSE 8888
 ---> Using cache
 ---> 7fcdbaf0fe40
Step 17/54 : ENV APP_BASE /srv
 ---> Using cache
 ---> ee51ea8ec11a
Step 18/54 : ENV NPM_DIR ${APP_BASE}/npm
 ---> Using cache
 ---> f34fdbe18446
Step 19/54 : ENV NPM_CONFIG_GLOBALCONFIG ${NPM_DIR}/npmrc
 ---> Using cache
 ---> 833e25d9d235
Step 20/54 : ENV CONDA_DIR ${APP_BASE}/conda
 ---> Using cache
 ---> 8ee468f15616
Step 21/54 : ENV NB_PYTHON_PREFIX ${CONDA_DIR}/envs/notebook
 ---> Using cache
 ---> 971c9391c0ab
Step 22/54 : ENV KERNEL_PYTHON_PREFIX ${NB_PYTHON_PREFIX}
 ---> Using cache
 ---> 7fa50abe8e21
Step 23/54 : ENV PATH ${NB_PYTHON_PREFIX}/bin:${CONDA_DIR}/bin:${NPM_DIR}/bin:${PATH}
 ---> Using cache
 ---> eaa621b5d97f
Step 24/54 : COPY build_script_files/-2fusr-2flocal-2flib-2fpython3-2e8-2fdist-2dpackages-2frepo2docker-2fbuildpacks-2fconda-2factivate-2dconda-2esh-1be18e /etc/profile.d/activate-conda.sh
 ---> Using cache
 ---> 9ffab983afa0
Step 25/54 : COPY build_script_files/-2fusr-2flocal-2flib-2fpython3-2e8-2fdist-2dpackages-2frepo2docker-2fbuildpacks-2fconda-2fenvironment-2epy-2d3-2e6-2efrozen-2eyml-7faaa9 /tmp/environment.yml
 ---> Using cache
 ---> 8bf90b0af9a4
Step 26/54 : COPY build_script_files/-2fusr-2flocal-2flib-2fpython3-2e8-2fdist-2dpackages-2frepo2docker-2fbuildpacks-2fconda-2finstall-2dminiconda-2ebash-813f10 /tmp/install-miniconda.bash
 ---> Using cache
 ---> d94262206d7f
Step 27/54 : RUN mkdir -p ${NPM_DIR} && chown -R ${NB_USER}:${NB_USER} ${NPM_DIR}
 ---> Using cache
 ---> a0d9a2009de8
Step 28/54 : USER ${NB_USER}
 ---> Using cache
 ---> d346cab5791b
Step 29/54 : RUN npm config --global set prefix ${NPM_DIR}
 ---> Using cache
 ---> d9689a512ce1
Step 30/54 : USER root
 ---> Using cache
 ---> 0698c74eac04
Step 31/54 : RUN bash /tmp/install-miniconda.bash && rm /tmp/install-miniconda.bash /tmp/environment.yml
 ---> Using cache
 ---> 923c36d4009c
Step 32/54 : ARG REPO_DIR=${HOME}
 ---> Using cache
 ---> be7951594e5a
Step 33/54 : ENV REPO_DIR ${REPO_DIR}
 ---> Using cache
 ---> dd36de80f9ae
Step 34/54 : WORKDIR ${REPO_DIR}
 ---> Using cache
 ---> 147d1c57f768
Step 35/54 : ENV PATH ${HOME}/.local/bin:${REPO_DIR}/.local/bin:${PATH}
 ---> Using cache
 ---> ee1cce3f64fc
Step 36/54 : ENV CONDA_DEFAULT_ENV ${KERNEL_PYTHON_PREFIX}
 ---> Using cache
 ---> f42b790c0d6f
Step 37/54 : COPY src/Pipfile ${REPO_DIR}/Pipfile
 ---> 90705c47d1a3
Step 38/54 : COPY src/Pipfile.lock ${REPO_DIR}/Pipfile.lock
 ---> 3b65cdc995da
Step 39/54 : USER root
 ---> Running in e37e7cabd338
Removing intermediate container e37e7cabd338
 ---> e85f4012398b
Step 40/54 : RUN chown -R ${NB_USER}:${NB_USER} ${REPO_DIR}
 ---> Running in 29189b9b23f6
Removing intermediate container 29189b9b23f6
 ---> 4ad7f0a54c52
Step 41/54 : USER ${NB_USER}
 ---> Running in ba34060777a4
Removing intermediate container ba34060777a4
 ---> b5e4aab56895
Step 42/54 : RUN ${KERNEL_PYTHON_PREFIX}/bin/pip install pipenv==2018.11.26
 ---> Running in 7c5d3c484d77
[91mWARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
[0mCollecting pipenv==2018.11.26
  Downloading pipenv-2018.11.26-py3-none-any.whl (5.2 MB)
Requirement already satisfied: pip>=9.0.1 in /srv/conda/envs/notebook/lib/python3.6/site-packages (from pipenv==2018.11.26) (20.0.2)
Collecting virtualenv-clone>=0.2.5
  Downloading virtualenv_clone-0.5.4-py2.py3-none-any.whl (6.6 kB)
Requirement already satisfied: setuptools>=36.2.1 in /srv/conda/envs/notebook/lib/python3.6/site-packages (from pipenv==2018.11.26) (45.1.0.post20200119)
Requirement already satisfied: certifi in /srv/conda/envs/notebook/lib/python3.6/site-packages (from pipenv==2018.11.26) (2019.11.28)
Collecting virtualenv
  Downloading virtualenv-20.0.27-py2.py3-none-any.whl (4.9 MB)
Collecting appdirs<2,>=1.4.3
  Downloading appdirs-1.4.4-py2.py3-none-any.whl (9.6 kB)
Requirement already satisfied: importlib-metadata<2,>=0.12; python_version < "3.8" in /srv/conda/envs/notebook/lib/python3.6/site-packages (from virtualenv->pipenv==2018.11.26) (1.5.0)
Collecting distlib<1,>=0.3.1
  Downloading distlib-0.3.1-py2.py3-none-any.whl (335 kB)
Requirement already satisfied: six<2,>=1.9.0 in /srv/conda/envs/notebook/lib/python3.6/site-packages (from virtualenv->pipenv==2018.11.26) (1.14.0)
Collecting filelock<4,>=3.0.0
  Downloading filelock-3.0.12-py3-none-any.whl (7.6 kB)
Collecting importlib-resources>=1.0; python_version < "3.7"
  Downloading importlib_resources-3.0.0-py2.py3-none-any.whl (23 kB)
Requirement already satisfied: zipp>=0.5 in /srv/conda/envs/notebook/lib/python3.6/site-packages (from importlib-metadata<2,>=0.12; python_version < "3.8"->virtualenv->pipenv==2018.11.26) (2.1.0)
Installing collected packages: virtualenv-clone, appdirs, distlib, filelock, importlib-resources, virtualenv, pipenv
Successfully installed appdirs-1.4.4 distlib-0.3.1 filelock-3.0.12 importlib-resources-3.0.0 pipenv-2018.11.26 virtualenv-20.0.27 virtualenv-clone-0.5.4
Removing intermediate container 7c5d3c484d77
 ---> 6cc8d167e86a
Step 43/54 : USER root
 ---> Running in b0b82dd9f7e8
Removing intermediate container b0b82dd9f7e8
 ---> c7dd3f5cd89b
Step 44/54 : COPY src/ ${REPO_DIR}
 ---> e9546ab03b76
Step 45/54 : RUN chown -R ${NB_USER}:${NB_USER} ${REPO_DIR}
 ---> Running in 7467c1e5b129
Removing intermediate container 7467c1e5b129
 ---> e80d5fb5a6c8
Step 46/54 : USER ${NB_USER}
 ---> Running in 6e52088b4846
Removing intermediate container 6e52088b4846
 ---> 7f576845770e
Step 47/54 : RUN (cd . &&                     PATH="${KERNEL_PYTHON_PREFIX}/bin:$PATH"                         pipenv install --ignore-pipfile --system --dev                 )
 ---> Running in 245131d650f2
Installing dependencies from Pipfile.lock (26b409)â€¦
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
Ignoring appnope: markers 'sys_platform == "darwin"' don't match your environment
Looking in indexes: https://pypi.python.org/simple
[91mAn error occurred while installing terminado==0.8.1 --hash=sha256:55abf9ade563b8f9be1f34e4233c7b7bde726059947a593322e8a553cc4c067a --hash=sha256:65011551baff97f5414c67018e908110693143cfbaeb16831b743fe7cad8b927! Will try again.
[0mInstalling initially failed dependenciesâ€¦
[91m[pipenv.exceptions.InstallError]:   File "/srv/conda/envs/notebook/lib/python3.6/site-packages/pipenv/core.py", line 1874, in do_install
[0m[91m[pipenv.exceptions.InstallError]:       keep_outdated=keep_outdated
[0m[91m[pipenv.exceptions.InstallError]:   File "/srv/conda/envs/notebook/lib/python3.6/site-packages/pipenv/core.py", line 1253, in do_init
[0m[91m[pipenv.exceptions.InstallError]:       pypi_mirror=pypi_mirror,
[0m[91m[pipenv.exceptions.InstallError]:   File "/srv/conda/envs/notebook/lib/python3.6/site-packages/pipenv/core.py", line 859, in do_install_dependencies
[0m[91m[pipenv.exceptions.InstallError]:       retry_list, procs, failed_deps_queue, requirements_dir, **install_kwargs
[0m[91m[pipenv.exceptions.InstallError]:   File "/srv/conda/envs/notebook/lib/python3.6/site-packages/pipenv/core.py", line 763, in batch_install
[0m[91m[pipenv.exceptions.InstallError]:       _cleanup_procs(procs, not blocking, failed_deps_queue, retry=retry)
[0m[91m[pipenv.exceptions.InstallError]:   File "/srv/conda/envs/notebook/lib/python3.6/site-packages/pipenv/core.py", line 681, in _cleanup_procs
[0m[91m[pipenv.exceptions.InstallError]:       raise exceptions.InstallError(c.dep.name, extra=err_lines)
[0m[91m[pipenv.exceptions.InstallError]: ['Looking in indexes: https://pypi.python.org/simple', 'Collecting terminado==0.8.1', '  Using cached terminado-0.8.1-py2.py3-none-any.whl (33 kB)', 'Installing collected packages: terminado', '  Attempting uninstall: terminado', '    Found existing installation: terminado 0.8.3']
[0m[91m[pipenv.exceptions.InstallError]: ['WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.', 'Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.', "To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.", "ERROR: Cannot uninstall 'terminado'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall."]
[0m[91mERROR: ERROR: Package installation failed...
[0mRemoving intermediate container 245131d650f2
The command '/bin/sh -c (cd . &&                     PATH="${KERNEL_PYTHON_PREFIX}/bin:$PATH"                         pipenv install --ignore-pipfile --system --dev                 )' returned a non-zero code: 1Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/repo2docker_checker/checker.py", line 324, in test_one_repo
    image, checkout_path = build_repo(
  File "/usr/local/lib/python3.8/dist-packages/repo2docker_checker/checker.py", line 128, in build_repo
    run(
  File "/usr/lib/python3.8/subprocess.py", line 512, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['jupyter-repo2docker', '--no-run', '--no-clean', '--image-name', 'r2d-test-github.com-agconti-dog-walks:8b6c692', '/tmp/r2d-test-2020-07-23T11.46jc68w0o1/github.com/agconti/dog-walks']' returned non-zero exit status 1.
